onnxruntime/python/tools/kernel_explorer/kernels/gemm.cc

```diff
@@ -31,3 +32,4 @@ void InitGemm(py::module mod) {
 }
 
 }  // namespace onnxruntime
+#endif
```

Comment:
No special includes, etc needed for `USE_CUDA`?

Reply:
Moved, not `#if` is needed is needed anymore.